### PR TITLE
[4.0] com_templates sidebar

### PR DESCRIPTION
**Pull request by brianteeman**
Removes the sidebar for templates and styles. They are not replaced as they are accessed directly from the system dashboard instead of the components menu

### CODE REVIEW

File: administrator/components/com_templates/View/Styles/HtmlView.php [hunk 1]
```diff
@@ -147,7 +147,5 @@ protected function addToolbar()
 		}
 
 		ToolbarHelper::help('JHELP_EXTENSIONS_TEMPLATE_MANAGER_STYLES');
-
-		\JHtmlSidebar::setAction('index.php?option=com_templates&view=styles');
```
Comment (Quy) [inline administrator/components/com_templates/View/Styles/HtmlView.php:151]: Delete line 99 in this file?

**Comment by infograf768** [comment]
Just taking off the sidebar is not so nice.
If one edits a style or modifies a template file, then it is necessary to display again the System Cpanel to go to the other template manager.
New Toolbar buttons may help.

**Comment by brianteeman** [comment]
Exactly as menitoned by @wilsonge in another PR and as I replied - that is beyond the scope of this pr and is perhaps something the template team have resolved already as the system dashboard is there idea

**Comment by Bodge-IT** [comment]
I have tested this item :red_circle: unsuccessfully on 4f60f73ff85a745311c75a2ccdee5393728ad61d<br /><br />Cleared cache after patch applied.
Checked all System & Component links items and found sidebar in:
* Clear Cache
* Clear Expired Cache
* Glocal
* Warnings
* Discover
* Install Language
* Extensions
* Update Sources
* Text Filters
* Settings (Duplicates Global)
Components:
* Smart Search
* Contacts
*  - Categories

Templates sidebar was removed.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24092">issues.joomla.org/tracker/joomla-cms/24092</a>.</sub>

**Comment by Quy** [comment]
@Bodge-IT This PR is for Templates sidebar.

**Comment by brianteeman** [comment]
@Bodge-IT As the pr says - this only removes the sidebar for templates and styles - not anything else. They all have their own prs

**Comment by Bodge-IT** [comment]
OK I missed that.  I'll retest.

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24092">issues.joomla.org/tracker/joomla-cms/24092</a>.</sub>

**Comment by softforge** [comment]
I have tested this item :white_check_mark: successfully on 4f60f73ff85a745311c75a2ccdee5393728ad61d<br /><br />Tested and removes the sidebar in style and template in both site and administrator<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24092">issues.joomla.org/tracker/joomla-cms/24092</a>.</sub>

**Comment by Bodge-IT** [comment]
I have tested this item :white_check_mark: successfully on 4f60f73ff85a745311c75a2ccdee5393728ad61d<br /><br />Sidebar removed for Styles and Templates on both Site and Admin
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24092">issues.joomla.org/tracker/joomla-cms/24092</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**